extensions/wikia/AdEngine/js/slot/inContent.js

```diff
@@ -14,7 +15,7 @@ define('ext.wikia.adEngine.slot.inContent', [
 	 * Adds dynamically new slot in the right place and sends tracking data
 	 */
 	function init(slotName) {
-		var adHtml = '<div id="' + slotName + '" class="wikia-ad default-height"></div>',
+		var adHtml = '<div id="INCONTENT_WRAPPER"><div id="' + slotName + '" class="wikia-ad default-height" data-label="' + msg('adengine-advertisement') + '"></div></div>',
```

Comment:
Not sure if it's worth since in order to use mustache template I would need to load it via ajax call